### PR TITLE
Cache 1000 genomes index in JSON file for easier quickstart

### PR DESCRIPTION
**Pull request by bfcrampton**

### CODE REVIEW

File: dataset_config/1000_genomes/README.md [hunk 1]
```diff
@@ -5,3 +5,14 @@ For a live Data Explorer of this dataset, see https://test-data-explorer.appspot
...
 (`ui.json` is here because it is only used by this repo.)
+
+For convenience, this repo reads the index from a JSON file. This way, one can
+quickly try out Data Explorer without having to set up and run any
+of the [data explorer indexers](https://github.com/DataBiosphere/data-explorer-indexers).
+
+### Update index JSON
+Ensure the elastic search index is already running at `http://localhost:9200`.
+```
+pip install elasticsearch
```
Comment (melissachang) [inline dataset_config/1000_genomes/README.md]: virtualenv ~/virtualenv/elasticsearch
source ~/virtualenv/elasticsearch/bin/activate
pip install elasticsearch
python dataset_config/1000_genomes/dump_index.py
deactivate

Comment (bfcrampton) [reply dataset_config/1000_genomes/README.md]: Done.

**Comment by bfcrampton** [comment]
Removed line in readme about test_dataset.

File: README.md [hunk 2]
```diff
@@ -16,7 +16,7 @@ and [here](https://github.com/DataBiosphere/data-explorer-indexers/tree/master/d
 
 ## Quickstart
 
-Run local Data Explorer with a test dataset:
+Run local Data Explorer with the 1000 Genomes dataset:
```
Comment (melissachang) [inline README.md]: This comment is actually for .gitignore

- Remove lines 4-6
- Have .gitignore ignore 1000_genomes/deploy.json. In general, deploy.json shouldn't be checked in. People can deploy to whatever project they want.

Also, in dataset_config/1000_genomes/README.md, remove "See here for other...".

Comment (bfcrampton) [reply README.md]: Done.

**Pull request merged**